packages/extension-vscode/scripts/bundle-size.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require('path');
+const packageJSON = require('../package.json');
+
+const filename = `${path.resolve(__dirname, '..')}/vscode-webhint-${packageJSON.version}.vsix`;
+
+fs.stat(filename, (err, stats) => {
+    if (err) {
+        throw new Error(`Reading bundle failed: ${err}`);
+    }
+
+    const sizeKB = Math.round(stats.size / 1000);
```

Comment:
Nit: Shouldn't this be `const sizeKB = Math.round(stats.size / 1024);`?

Reply:
Either that or a lower-case "k" according to wikipedia. https://en.wikipedia.org/wiki/Kilobyte

Honestly doesn't matter much in this case as we're using this just to keep tabs on bundle size growth. I went for consistency with what we were already doing for the browser extension.

Reply:
hehehe, then `1000` is ok :)